packages/lib/tracing/index.ts

```diff
@@ -0,0 +1,95 @@
+export interface TraceContext {
+  traceId: string;
+  spanId: string;
+  parentSpanId?: string;
+  operation: string;
+  // So that we don't violate open closed principle, we allow meta to be added to the trace context
+  meta?: Record<string, any>;
```

Comment:
Let's keep it strict like string only maybe? Ideally we shouldn't have too uch data in meta as that is prefixed in all the messages, so string restriction seems good to me.